src/ServiceControl/CompositeViews/Messages/GetMessagesByQuery.cs

```diff
@@ -22,7 +22,6 @@ public GetMessagesByQuery()
             Get["/messages/search/{keyword*}"] = parameters =>
             {
                 string keyword = parameters.keyword;
-                keyword = keyword?.Replace("/", @"\");
```

Comment:
Note how the equivalent case for searching within a specific endpoint [does not do this replacement](https://github.com/Particular/ServiceControl/blob/10f82ebab164c62833fd3c71f9ead21ff346955e/src/ServiceControl/CompositeViews/Messages/GetMessagesByQuery.cs#L37-L43).